projects/js-packages/connection/components/disconnect-dialog/steps/step-disconnect.jsx

```diff
@@ -52,6 +52,7 @@ const StepDisconnect = props => {
 				disabled={ isDisconnecting }
 				onClick={ onDisconnect }
 				className="jp-connection__disconnect-dialog__btn-disconnect"
+				ariaLive="polite" // Announce "Disconnecting" when this button is pressed
```

Comment:
I'm wondering if `ariaLive="assertive"` would be better in this case. In this step, both `polite` and `assertive` would work similarly from the website perspective since we are not taking any other action at the same time, but I think if the screen reader reads another message at the same time, we should prioritize the "Disconnecting..." message. What do you think?

Reply:
> I'm wondering if ariaLive="assertive" would be better in this case

Hmmm... I think there's a case for using `assertive` here. Typically I tend to avoid using `assertive`, since it can be really disruptive, but I think we would want to announce that the site was "Disconnecting" since missing that information could make the experience of using the screen reader less clear (if the user didn't know they had successfully triggered the disconnect action).